javascript/ui/page/passwordsignup.js

```diff
@@ -43,6 +44,7 @@ goog.require('firebaseui.auth.ui.page.Base');
  */
 firebaseui.auth.ui.page.PasswordSignUp = function(
```

Comment:
Sounds great! Thanks for the pointer. This is definitely a much cleaner way to handle that case. 👍